daffodil-test/src/test/scala/org/apache/daffodil/section23/dfdl_expressions/TestDFDLExpressions.scala

```diff
@@ -516,10 +516,11 @@ class TestDFDLExpressions {
   //@Test def test_namespace_uri_01() { runner2.runOneTest("namespace_uri_01") }
   //@Test def test_namespace_uri_02() { runner2.runOneTest("namespace_uri_02") }
   //DFDL-1114
-  //@Test def test_namespace_uri_03() { runner2.runOneTest("namespace_uri_03") }
```

Comment:
Are changes still needed to get 01 and 02 to pass?

Reply:
Yes, we need additional changes to fix DAFFODIL-1101 before 01 and 02 will pass.  I already have another branch with these changes made and tested but I will make a second pull request after this one to make these additional changes easier to review.